backend/app/routes.py

```diff
@@ -238,6 +238,13 @@ def aggregate_travel_times(start_node, end_node, start_time, end_time, start_dat
         FROM corridor_period_daily_avg_tt 
     '''
 
+    if incl_holiday:
+        holiday = '0,1'
+    else: 
+        holiday = '1'
+
+    print(tuple(dow_list))
+

```

Comment:
What values does the holiday param take? Aren't these also strings? Does 'true' evaluate to `True`? Does 'anyRandomValue' evaluate to `True`?